gepetuto/generate.py

```diff
@@ -20,8 +20,19 @@ def generate(tp_id: List[int], **kwargs):
 def generate_from_id(tp_id: int):
     """Find the corresponding ipynb and folder for a given tp_id."""
     folder = Path() / f"tp{tp_id}"
-    ipynb = next(Path().glob(f"{tp_id}-*.ipynb"))
-    generate_ipynb(ipynb, folder)
+    ipynbs = Path().glob(f"{tp_id}-*.ipynb")
+    for ipynb in ipynbs:
+        generate_ipynb(ipynb, folder)
+
+
+def fix_end_of_content(content):
+    """Remove extra lines."""
+    while 1:
+        if content[-1] == "\n":
+            content.pop()
+        else:
+            return content
+    return None
```

Comment:
This is basically a `content.strip()`, isn't it ? Or maybe a `content.strip('\n')` if you want to keep other white spaces than line breaks

Reply:
I'm not sure it's possible to use strip here, content is a list of string

Reply:
Ok, this is more clear, thanks